launcher/launcher_test.go

```diff
@@ -289,6 +289,7 @@ func (s *LauncherTestSuite) TestWaitForUpdates() {
 	}
 
 	for _, tt := range testcases {
+		tt := tt
```

Comment:
?

Reply:
Common golang idiom, the `tt` variable needs to be captured when used in goroutines.

See https://eli.thegreenplace.net/2019/go-internals-capturing-loop-variables-in-closures/

This is fixed in 1.22 but we are still not in that version.